data/merge_acs_data.py

```diff
@@ -0,0 +1,128 @@
+import pandas as pd
+import numpy as np
+
+# Read in two files - one, a population by geography file produced by NHGIS
+# The data can be found with these filters:
+# DATASETS : 	2015_2019_ACS5a
+# TOPICS : Total Population
+# The data should be downloaded at the block group level, the highest resolution geo data available from the ACS
+
+# The second file is our master block data set, which is produced elsewhere in this repository.
+# See 'create_master_block_dataset.py'
+
+
+acs = pd.read_csv(
```

Comment:
name this `block_group`, or `acs_block_group` to distinguish from `census_block_group` later (clearer to specify the unit of the data in the dataset name)